docs/learning/assets/learning-page.css

```
@@ -0,0 +1,397 @@
+  font-size: 1.08rem;
+}
+
+blockquote > :last-child {
+  margin-bottom: 0;
+}
+
+code {
+  padding: 0.12em 0.34em;
+  border-radius: 4px;
+  background: var(--panel-soft);
+  font-family: "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.1)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)